main.cpp

```diff
@@ -83,43 +105,91 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 
 /*
  1)
- */
+ */
+float bodyMassIndex(float height = 175.f, float weight = 75.f)
+ {
+    ignoreUnused(height, weight);
+    return {};
+ }

```

Comment:
Curly braces that define the start and end of a function should align with the word defining the return type (in this case: `float`)
- [ ] revise

This function name does not contain a verb/action
Function names describe work that is performed, that means they need to have verbs in their name
- [ ] revise